src/plugins/init/settings/screens/RevengePluginsSettingScreen.tsx

```diff
@@ -38,9 +50,15 @@ export default function RevengePluginsSettingScreen() {
         [],
     )
 
+    const sortedPlugins = useMemo(
```

Comment:
Don't sort plugins before filtering them. This makes it less efficient.
You can sort the plugins after they are filtered right in the other `useMemo`, simply rename the variable.